public/data/kumarsonsoff3.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Priyanshu Kumawat",
+  "type": "personal",
+  "bio": "Web developer | Tech writer | Open Source Enthusiast!",
+  "avatar": "https://github.com/kumarsonsoff3.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/kumarsonsoff3",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Kumar_Sons_off",
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Completed JavaScript Udemy course",
+      "date": "December 2021",
+      "icon": "globe",
+      "color": "green",
+      "description": "Completed JavaScript Udemy course by Jonas schmedtmann"
+    },
+    {
+      "title": "Started Freelancing",
+      "date": "November 2021",
+      "icon": "dollar",
+      "color": "gold",
+      "description": "Started freelancing"
+    },
+    {
+      "title": "Published Firt blog on Hashnode",
+      "date": "October 2021",
+      "icon": "hashnode",
+      "color": "blue",
+      "description": "Published First blog on Hashnode"
```

Comment:
```suggestion
      "description": "Published my first blog on Hashnode"
```

Reply:
Thanks for your contribution to this fixing typo!!